src/Esri.ArcGISRuntime.Toolkit.sln

```diff
@@ -59,12 +59,11 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Esri.ArcGISRuntime.Toolkit.
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Native", "Native", "{1D1309F0-3323-4686-A08E-EDEA9CF21070}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Toolkit.SampleApp.WPF_Net5", "Samples\Toolkit.SampleApp.WPF_Net5\Toolkit.SampleApp.WPF_Net5.csproj", "{35055508-4AEB-449A-900D-855F78E01933}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Toolkit.SampleApp.WPF_Net5", "Samples\Toolkit.SampleApp.WPF_Net5\Toolkit.SampleApp.WPF_Net5.csproj", "{35055508-4AEB-449A-900D-855F78E01933}"

```

Comment:
Did you mean to make changes to this file?

Reply:
Visual Studio insists on making this change and the next whenever I save. I have to be careful not to commit them into each PR where they aren't relevant. I wanted to get them merged now to avoid ongoing noise.

The lines adding deploy configurations for iPhone are intended to make the build/debug experience easier.